src/index.ts

```diff
@@ -43,7 +43,14 @@ export {RadixTrie, Leaf, eventToHotkeyString}
 export function install(element: HTMLElement, hotkey?: string): void {
   // Install the keydown handler if this is the first install
   if (Object.keys(hotkeyRadixTrie.children).length === 0) {
-    document.addEventListener('keydown', keyDownHandler)
+    const scopedElementId = element.getAttribute('data-hotkey-scope')
+    if (scopedElementId) {
+      const scopedElement = document.getElementById(scopedElementId)
+      if (!scopedElement) return
+      scopedElement.addEventListener('keydown', keyDownHandler)
+    } else {
+      document.addEventListener('keydown', keyDownHandler)
+    }
```

Comment:
I think we want to check if there is scope in the key down handler itself?

Conditionally installing an event handler like this means that if we install first on a scoped element, we'll never install the event handler on the document which means that global hotkeys won't be handled.